Packages/com.unity.inputsystem/InputSystem/Utilities/MemoryHelpers.cs

```diff
@@ -228,6 +226,93 @@ public static bool MemCmpBitRegion(void* ptr1, void* ptr2, uint bitOffset, uint
             return true;
         }
 
+        public static void MemSet(void* destination, int numBytes, byte value)
```

Comment:
Wow, surprised this does not exist (well, it does, Unsafe.InitBlock, but only in .NET Core). Guess there is the Initblk opcode, but no C# code to generate it. Weird.